server/controllers/userController.js

```diff
@@ -0,0 +1,39 @@
+import database from "../models/index";
+
+import controller from "../helpers/index";
+
+const { comparePassword, generateJwtToken, httpResponse } = controller;
```

Comment:
What is generateJwtToken, why not generateJWT, as the former is somehow repetitive